config/routes.rb

```diff
@@ -1,3 +1,14 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  mount Rswag::Ui::Engine => '/api-docs'
+  mount Rswag::Api::Engine => '/api-docs'
+
+  devise_for :users, controllers: {
+    sessions: 'users/sessions',
+    registrations: 'users/registrations'
+  }
+
+  namespace :api do
+    resources :reservations
+    resources :items, path: '/dances'
+  end
```

Comment:
- [ ] I recommend that you specify which routes you will be using here instead of creating all of them, this way you won't have unused ones.

Reply:
Done!